can_tools/scrapers/official/NC/nc_vaccine.py

```diff
@@ -15,7 +17,7 @@ class NCVaccine(TableauDashboard):
     location_type = "county"
     baseurl = "https://public.tableau.com"
     viewPath = "NCDHHS_COVID-19_Dashboard_Vaccinations/Summary"
-
+    location_name = "NorthCarolina"
```

Comment:
The name in the database is `North Carolina` This will cause the problem with the "missing `location_id`" error. 

However, I changed it locally and I'm still getting the error. Still investigating.

Reply:
Any thoughts here @tlyon3 on how best I should handle this?

Reply:
Just rename this line to `location_name = 'North Carolina'`

Reply:
The issue is still coming up. This is correct though. Must be a bug in the `put` method. @sglyon, do you have any insight as to why this would be failing? 

```
In [26]: d.find_unknown_location_id(engine, df)
Out[26]:
Empty DataFrame
Columns: [vintage, dt, category, measurement, unit, age, race, ethnicity, sex, value, location_name]
Index: []

In [27]: df.location_name.unique()
Out[27]: array(['North Carolina'], dtype=object)
```

Reply:
see below about `location_type = "state"`